### DISCUSSION TITLE
English Translation / abbreviation for the month of May

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1356] [General]
In the Sensor Chart and Statistics Chart using English the month of May is shown as "Mai".
Tour Book Date correctly shows "May"

**Comment by wolfgang-ch** [comment]
This bug is fixed in https://github.com/mytourbook/mytourbook/commit/ff0993019d9300571e497c935b4eacea0952a438 it exist since the beginning in 2007

![image](https://github.com/mytourbook/mytourbook/assets/1283445/d9bc277f-0abe-4c26-a636-a6d11408ef7e)

**Comment by FJBDev** [comment]
Good catch @stvzldn !
